lib/index.js

```diff
@@ -1 +1,6 @@
-module.exports = require('./key-handler');
+/* @flow */
+import KeyHandler from './key-handler';
+export {KEYDOWN, KEYPRESS, KEYUP} from './constants';
+export {default as keyHandleDecorator, keyToggleHandler, keyHandler} from './key-handle-decorator';
+
+export default KeyHandler;
```

Comment:
Won't the end-user now have to use something like webpack? (vs. when using `module.exports`)

Reply:
I don't think so but if we go with rollup it's easy to deliver umd, es6, cjs and other packages

Reply:
Rollup definitely wasn't as popular as it is now when this package was initially released, seems most libraries moved to it, and sounds like we should too

Reply:
Sure, I will prepare a PR for it once we're done with this. The docs make it sound like it should be super easy but then you start looking into all the options and its a :rabbit2: :hole: I have a good setup in a similar project although in that one I used Typescript so it needs some tweaking before we can deploy it.

I'm not a great fan but I think webpack-dev-server is super convenient for the demo and therefore we may want to keep it in place.